auth-web/src/components/auth/BcscPanel.vue

```diff
@@ -56,13 +64,14 @@ import { User } from '@/models/user'
   }
 })
 export default class BcscPanel extends Vue {
+  private cardSetUpUrl = 'https://www2.gov.bc.ca/gov/content/governments/government-id/bc-services-card/log-in-with-card/mobile-card/set-up-mobile-card'
+  private learnMoreUrl = 'https://www2.gov.bc.ca/gov/content/governments/government-id/bc-services-card/log-in-with-card/mobile-card'
```

Comment:
If you feel REALLY type-y, these could even be `private readonly ...`

Reply:
Ah! Will update this for sure.

Reply:
Updated in next commit!